enarx-keep/tests/bin/read.c

```diff
@@ -0,0 +1,22 @@
+#include "libc.h"
+
+int main(void) {
+    int min(int a, int b) { return a < b ? a : b; }
+
+    char buf[12] = {'\0'};
```

Comment:
This initializer is a bit strange because of what is actually happening. Your initializer is roughly "zero the first byte." Fortunately, C specifies what should happen when some bytes are left unspecified: they are zeroed. Since you're relying on that behavior anyway, it is simpler and cleaner to just do:

```suggestion
    char buf[12] = {};
```

Also, I'm not sure why `12` is used as the size here since it isn't a power of two. But maybe there is a reason.

Reply:
It was the size of the current string. So, a reason, but not a good reason. : )
Didn't know that about initializing, either.